src/openexr.imageio/exrinput.cpp

```diff
@@ -113,9 +113,31 @@ static std::map<std::string, std::string> exr_tag_to_oiio_std {
     { "pixelAspectRatio", "PixelAspectRatio" },
     { "xDensity", "XResolution" },
     { "expTime", "ExposureTime" },
+    { "aperture", "FNumber" },
+    { "isoSpeed", "Iso", },
+    { "focus", "Focus", },
+    { "cameraCCTSetting", "openexr:CameraTemperature", },
+    { "cameraTintSetting", "openexr:CameraTint", },
+    { "nominalFocalLength", "NominalFocalLength", },
+    { "pinholeFocalLength", "PinholeFocalLength", },
+    { "effectiveFocalLength", "EffectiveFocalLength", },
+    { "entrancePupilOffset", "EntrancePupilOffset", },
+    { "sensorPhotositePitch", "SensorPhotositePitch", },
```

Comment:
Why are these, and many like it, in the table at all?

What is the point in declaring a new OIIO standard metadata called "effectiveFocalLength" that is automatically translated to an OpenEXR attribute called "EffectiveFocalLength"?

Remember what I explained earlier:

The table is for items that are "standard" in the sense that OIIO-using applications already have a name for them that is intended to be the same across all image file format types, *AND* OpenEXR conventions expect the very same item to have a different name. We resolve this discrepancy by automatically translating the names between the OIIO conventional name and the OpenEXR conventional name, using these tables.

* Any item that isn't an "OIIO standard metadata name" should not be in the table.
* Any item that is an OIIO standard metadata name, and the EXR name is the same spelling, also doesn't need to be in the table.

Remember that OpenEXR already has the ability to store arbitrary name+value metadata without anything special being done, and without needing to explicitly list the set of names. None of this is necessary for OIIO simply to read and write these metadata in OpenEXR files. It already does that. If you tell OIIO (today, before your patch) to write a metadata called "EffectiveFocalLength" to an OpenEXR file, it will do that with no problem whatsoever.

Reply:
done.